app/services/hyrax/autopopulation/create_file.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module Autopopulation
+    class CreateFile
+      attr_accessor :url, :work, :user, :account, :filename
+
+      # params :url is the remote file localtion from unpaywall response
+      #
+      def initialize(url, user, work = nil, account = nil)
+        AccountElevator.switch!(account.cname) if Rails.application.config.hyrax_autopopulation.storage_type == "activerecord"
+
+        @url = url&.strip
```

Comment:
You can combine these lines: 

```ruby 
        return if (@url = url&.strip).blank?
```

Its also best to avoid unless if you have the option, `if blank?` is easier to reason.